pkg/array/array.go

```diff
@@ -254,30 +254,54 @@ func GetPowerStoreArrays(fs fs.Interface, filePath string) (map[string]*PowerSto
 	return arrayMap, mapper, defaultArray, nil
 }
 
-// ParseVolumeID parses volume id in from CO (Kubernetes) and tries to understand what in it are PowerStore volume id, and what is ip, protocol.
-// "/" is used as a delimiter.
+// ParseVolumeID parses a volume id from the CO (Kubernetes) and tries to extract the PowerStore volume id, Global ID, and protocol.
 //
 // Example:
 //
 //	ParseVolumeID("1cd254s/192.168.0.1/scsi") will return
```

Comment:
Actually, this is one of the supported legacy formats. Give it an IP and it should return the global ID.
I will update the example var names tho.

Reply:
I don't think we should still support this legacy format. Not sure if such upgrade paths are supported.

Reply:
It would be nice to get away from the legacy format, but I'm also unsure. We could potentially add a k8s job workload that validates and updates existing volume names, but it would be a non-trivial request.